src/pages/TeamPage/index.js

```diff
@@ -285,67 +284,65 @@ function TeamPage() {
                       (condition === "teamMembers" &&
                         item.Association !== "Volunteer")
                     ) {
-                      if (item.Content.length !== null)
-                        return (
-                          <Grid item xs={6} sm={6} md={3}>
-                            <Tippy
-                              animation="fade"
-                              interactive="true"
-                              duration={[500, 0]}
-                              placement={
-                                window.screen.availWidth < 650
-                                  ? "bottom"
-                                  : "right"
-                              }
-                              content={
-                                <Popup
-                                  Name={item.Name || "Awaiting Member's Name"}
-                                  Content={
-                                    (item.Content.length && item.Content) ||
-                                    "Awaiting content from team member"
+                      return (
+                        <Grid item xs={6} sm={6} md={3}>
+                          <Tippy
+                            animation="fade"
+                            interactive="true"
+                            duration={[500, 0]}
+                            placement={
+                              window.screen.availWidth < 650
+                                ? "bottom"
+                                : "right"
+                            }
+                            content={
+                              <Popup
+                                Name={item.Name || "Awaiting Member's Name"}
+                                Content={
+                                  (item.Content.length && item.Content) ||
+                                  "Awaiting content from team member"
```

Comment:
1. If `item.Content.length` then it will display the content `item.Content`.  That's why it has written `item.Content.length && item.Content`
2. If the content has length `item.Content.length` then it will display the content `item.Content` or else it will display the string.
I didn't get 3 & 4. What are you talking about?

Reply:
1.  Sorry, disregard this point, I was focusing on the possibility of the errors that would occur if `item.Content` is `null` or `undefined`, which we're assuming won't happen.
2. My point here is that you're also referring to the hard-coded string in the below for styling so if you decide to change the message that's shown on line 303 (to something like "Loading content", you'd also have to remember to change it below or this text wouldn't appear as grey): https://github.com/navgurukul/bhanwari-devi/blob/5c16f440990690ea5eca00d386fcfd2f1d693563/src/pages/TeamPage/index.js#L107-L111
My recommendation here would be to pass `item.Content` (regardless of whether it's an empty Array or not), and then assign a constant like `const isContentLoading = props.Content.filter(item => item.trim()).length === 0`.  Then you could replace the above lines with `style={isContentLoading ? {color: grey} : {}}`  
3. With the current setup, if `item.Content.length` is `0`, then `props.Content` will be a **string** "Awaiting content from team member".  If its length is greater than 0, then `props.Content` will be an Array.  So `props.Content` can be a `string` or an `Array`.  What this means is that if the description were say an Array of lines and you wanted it to appear with line breaks in the HTML, you might do something like this: `props.Content.join('<br />')`.  But if `props.Content` can also be the string "Awaiting content from team member", this would cause the code to crash as `join` cannot be called on a string.  You could always add an `if` statement to check that it's an Array, but that will unnecessarily complicate the code.
4. If you did something like this:
`const Msg = {};`
And in en.js, you had an object assignment like this:
Msg.EN = {
  "TEAM_HEADING": "Meraki Team",
  "AWAITING_CONTENT": "Awaiting content from team member",
};
And in hi.js, you had an object assignment like this:
MSG.HI = {
  "TEAM_HEADING": "मेराकी टीम",
  "AWAITING_CONTENT": "(Hindi translation for Awaiting content from team member)",
};

Then you could write `Msg.currentLanguage.TEAM_HEADING` or `Msg.currentLanguage.AWAITING_CONTENT` and when you switch assignments from `Msg.currentLanguage = Msg.EN;` to say `Msg.currentLanguage = Msg.HI`, the text would all switch from English to Hindi.

Reply:
Actually, since you're filtering out empty content Arrays here:
https://github.com/navgurukul/bhanwari-devi/blob/cd36d5f8e04585d2606e7b57fc66ef328722fdd7/src/pages/TeamPage/index.js#L62
will it even be possible to see `"Awaiting content from team member"` now?  Sorry if I'm asking questions about stuff that will change later.

Reply:
No worries. That's absolutely fine. No it won't show this content now.